docs/Converters/UKBtoBIDS.md

```diff
@@ -1,70 +1,89 @@
 <!-- markdownlint-disable MD046 -->
 # `ukb-to-bids` – Conversion of the UK Biobank (UKB) to BIDS
 
-!!! quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
+??? quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
     UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine it and has led to the discovery of several scientific advances and numerous treatments to improve human health.
 
-## Dependencies
+## Downloading the Data
 
-If you installed the core of Clinica, this converter needs the `dcm2niix` package.
+The UKB to BIDS converter assumes that the user has already got access to the data and has downloaded both imaging and clinical data locally.
 
-## Downloading UK Biobank
+!!! danger "Organising data with the aim of using the converter"
+    Be careful, the file {==clinical_data.csv==}  needs to contain the following information about the subject :
+    *sex, year of birth, age at recruitment, age at sessions*.
+    There are two columns as there are two imaging sessions.
+    The names of the columns should be kept as they are when downloaded using `ukbconv` using options `csv`.
 
-The UK Biobank to BIDS converter assumes that the user has already applied to get access to the data and that the imaging and clinical data have already been downloaded locally.
 
-The imaging data can be downloaded and extracted using [this tutorial](https://biobank.ctsu.ox.ac.uk/~bbdatan/Accessing_UKB_data_v2.3.pdf). The clinical data can be extracted using the information from [this repository](https://github.com/kenhanscombe/ukbtools).
 
-Be careful, the file `clinical_data.csv` needs to contain the following information about the subject: sex, year of birth, age at recruitment, age at sessions (there are two columns as there are two imaging sessions). The names of the columns should be kept as they are when downloaded using `ukbconv` using options `csv`.
+## Using the converter
+### Dependencies
 
-## Supported modalities
+If you [installed the core of Clinica](../Installation.md#install-clinica), this converter needs the [dcm2niix]((../Third-party.md#converters)) package.
```

Comment:
The link for dcm2niix is broken. I think it's because of the double parenthesis:

```suggestion
If you [installed the core of Clinica](../Installation.md#install-clinica), this converter needs the [dcm2niix](../Third-party.md#converters) package.
```

Reply:
Oops, thanks for spotting it @NicolasGensollen !